### PR TITLE
Nette 2.3 compatibility

### PR DESCRIPTION
**Pull request by matej21**

### CODE REVIEW

File: src/Kdyby/Validator/Caching/Cache.php [hunk 1]
```diff
@@ -56,7 +56,7 @@ public function has($class)
 	 */
 	public function read($class)
 	{
-		return $this->has($class) ? $this->cache[$class] : FALSE;
+		return $this->cache->load($class) ?: FALSE;
```
Comment (Majkl578) [inline src/Kdyby/Validator/Caching/Cache.php]: What if it returns 0?

File: tests/KdybyTests/Validation/Extension.phpt [hunk 2]
```diff
@@ -34,7 +34,7 @@ class ExtensionTest extends Tester\TestCase
 		$config = new Nette\Configurator();
 		$config->setTempDirectory(TEMP_DIR);
 		$config->addParameters(array('container' => array('class' => 'SystemContainer_' . md5(time()))));
-		$config->addConfig(__DIR__ . '/../nette-reset.neon');
+		$config->addConfig(__DIR__ . '/../nette-reset.neon', \Nette\Framework::VERSION_ID >= 20300 ? 'v23' : 'v22');
```
Comment (Majkl578) [inline tests/KdybyTests/Validation/Extension.phpt]: Useless FQCN, Nette is in use.

File: src/Kdyby/Validator/Caching/Cache.php (hunk 1 above)
Comment (matej21) [inline src/Kdyby/Validator/Caching/Cache.php]: 0 is invalid value, see phpdoc of CacheInterface:
`@return ClassMetadata|false A ClassMetadata instance or false on miss`

Comment (Majkl578) [inline src/Kdyby/Validator/Caching/Cache.php]: fine

File: tests/KdybyTests/nette-reset.neon [hunk 3]
```diff
@@ -1,21 +1,26 @@
...
+v22 < common:
+	nette:
+		security:
+			frames: null
 
-nette:
-	security:
+		session:
+			autoStart: false
+v23 < common:
```
Comment (fprochazka) [inline tests/KdybyTests/nette-reset.neon]: I really like this approach, hope @dg won't discontinue this in the future...

Comment (dg) [inline tests/KdybyTests/nette-reset.neon]: Send test to nette ;)

Comment (fprochazka) [inline tests/KdybyTests/nette-reset.neon]: @dg deal!

**Comment by enumag** [comment]
ping @fprochazka - I'd like to have a new tagged version with Symfony 2.5 and Nette 2.3 support.

**Comment by fprochazka** [comment]
@enumag I'm aware of that :)

File: tests/KdybyTests/nette-reset.neon (hunk 3 above)
Comment (fprochazka) [inline tests/KdybyTests/nette-reset.neon]: @dg Actually, there are already tests for sections :)

**Comment by fprochazka** [comment]
Thank you, merged

**Pull request closed without merging**